ios-plugin/src/ZendeskProviderRequest.m

```diff
@@ -65,13 +65,13 @@ void _zendeskRequestProviderAddComment(char * gameObjectName, char * callbackId,
               withCallback:callback];
 }
 
-void _zendeskRequestProviderGetTicketFormWithIds(char * gameObjectName, char * callbackId, int ticketFormsIds[], int formsCount) {
+void _zendeskRequestProviderGetTicketFormWithIds(char * gameObjectName, char * callbackId, int64_t ticketFormsIds[], int formsCount) {
```

Comment:
There were some issues while using `long long`because of different implementations at different platform/architectures so i provide a specific integer size. `int32_t` cannot handle the current id values.